main.py

```diff
@@ -1,9 +1,13 @@
 from fastapi import FastAPI
 
-from .routes import users, login
+from routes import users
```

Comment:
Use `from .routes import users`

Reply:
this error occurs:
from .routes import users
ImportError: attempted relative import with no known parent package